alarm_backend/app.py

```diff
@@ -13,7 +16,32 @@
 
 
 @app.route("/alarms")
-def root():
+def get_all_alarms():
     """Get all alarms"""
+    all_alarm = []
```

Comment:
No bueno. Probably want to have a user concept so that we can store alarms per user.